Sources/LLVM/IRType.swift

```diff
@@ -36,6 +36,16 @@ public extension IRType {
   }
 }
 
+extension IRType {
+    internal func lowerFromVector() -> IRType {
```

Comment:
I don't anticipate this being used outside of the IRBuilder, so a private free function would work.

Reply:
Sure, I head home in a bit and then I can update the PR.